wfe2/wfe.go

```diff
@@ -421,33 +421,34 @@ func (wfe *WebFrontEndImpl) Handler(stats prometheus.Registerer, oTelHTTPOptions
 
 	// POSTable ACME endpoints
 	wfe.HandleFunc(m, newAcctPath, wfe.NewAccount, "POST")
-	wfe.HandleFunc(m, acctPath, wfe.Account, "POST")
+	wfe.HandleFunc(m, acctPath+"{acctID}", wfe.Account, "POST")
```

Comment:
I was wondering why you didn't just change `acctPath` to include the pattern, but then I realized that we also use `acctPath` (and similar) as an argument to `web.relativeEndpoint`. Is there a reasonable way to update relativeEndpoint to take a pattern as input?